CONTRIBUTING.md

```diff
@@ -169,23 +169,49 @@ build. Breaking the rules will result in a build failure.
 This repo is a great place to contribute a new instrumentation, exporter or any
 kind of extension. Please refer to [this
 page](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/extending-the-sdk/README.md#extending-the-opentelemetry-net-sdk)
-for help writing your component. Although the projects within this repo share
-some properties and configurations, they are built and released independently.
-So if you are creating a new project within `/src` and corresponding test
-project within `/test`, here are a few things you should do to ensure that your
-project is automatically built and shipped through CI.
+for help writing your component.
+
+When contributing a new project you are expected to assign either yourself or
+someone else who would take ownership of the component you are contributing. The
+owner should at least be an [OpenTelemetry
+Member](https://github.com/open-telemetry/community/blob/main/community-membership.md#member)
+to be eligible to assigned as component owner. Once the owner is identified,
```

Comment:
I think that it will be good to explain why it is important - possibility to auto-assign for PRs.